sys/net/gnrc/link_layer/gomach/gomach.c

```diff
@@ -1976,7 +1976,12 @@ static int _send(gnrc_netif_t *netif, gnrc_pktsnip_t *pkt)
 
 static void _gomach_msg_handler(gnrc_netif_t *netif, msg_t *msg)
 {
+    netdev_t *dev = netif->dev;
     switch (msg->type) {
+        case NETDEV_MSG_TYPE_EVENT:
+            dev->driver->isr(dev);
```

Comment:
1. Because the `NETDEV_MSG_TYPE_EVENT` was initially handled by `_gnrc_netif_thread` (in `gnrc_netif.c`). Now this was removed out of there in order to remove the hardcoded dependency to netdev.
2. The only reason is because this function requires more `msg->type`s that those provided by `gnrc_netif_msg_handler_netdev`. Extending that functions is not so elegant :(
3. Ditto

Reply:
ah, nevermind 2. I remember I did it anyway...